config/dev.exs

```diff
@@ -1,11 +1,11 @@
 use Mix.Config
 
+database_url =
+  System.get_env("DATABASE_URL") || "ecto://postgres:postgres@localhost/chat_api_dev"
+
 # Configure your database
 config :chat_api, ChatApi.Repo,
-  username: "postgres",
-  password: "postgres",
-  database: "chat_api_dev",
-  hostname: "localhost",
```

Comment:
@cheeseblubber will this break things for us?

Reply:
This is fine because its specified in the database URL